### ISSUE TITLE
Can't access process within preload script

### ISSUE THREAD

**Issue by CharlieHess** [atom/atom-shell#1039]
Within the `preload` script of a renderer process, I'm unable to access the `process` object without throwing an exception.

**Comment by zcbenz** [comment]
Can you run the following code in `preload` script and paste the result?

```js
console.log(window.location.href);
console.log(process);
console.log(module);
console.log(require);
```

**Comment by CharlieHess** [comment]
It's intermittent for me; as soon as I tried to log them I was able to access the `process` object. I'll try it next time it occurs.

**Comment by CharlieHess** [comment]
@zcbenz: Just tested this with your fix for #1038, it's working :100: percent now.
